blueprint/core/blueprint_ReactApplicationRoot.cpp

```diff
@@ -150,7 +150,9 @@ namespace blueprint
         DBG("");
         DBG(err.what());
 
-        errorText = std::make_unique<juce::AttributedString>(err.stack);
+        errorText = std::make_unique<juce::AttributedString>(err.what());
```

Comment:
A bit of an audacious change, but `EcmascriptEngine::Error`s thrown with no `stack` rendered nothing in the red error despite having a message to work with.  I'll see about the right way to get a stack / more context from failed invocations and undo this.